scripts/run_portserver.py

```diff
@@ -486,10 +487,13 @@ def handle_connection(
         """
         request = connection.recv(Server.message_size)
         response = handler(request)
-        connection.sendall(response)
+        # Here we use cast because response is of type socket and
```

Comment:
We shouldn't use cast. Why is there a mismatch here? Is one type a subtype of the other (in which case, maybe a type assertion is more appropriate)?